### PR TITLE
[for 2.6.x] Fix/2681 breadcrumbs consistency in workspace content

### PR DESCRIPTION
**Pull request by metzgegu**
#2681

## Checkpoints

The goal of this section is to help code reviewer to do required checks on this pull request. Please don't edit this list.
If one or more of these checkpoints are out of scope, please write below why they are.

- [x] Code is clear enough
- [x] If there are FIXME in the code, then there are associated issues and issue is referenced in the FIXME
- [x] If there are TODO, NOTE or HACK in code, date and developer initials are present
- [x] Automated tests have been written (covering feature or non regression), or *at least* an issue has been created for test implementation

- [x] Tested by quality team

### CODE REVIEW

File: frontend/src/container/WorkspaceContent.jsx [hunk 1]
```diff
@@ -263,6 +270,9 @@ class WorkspaceContent extends React.Component {
       })
     }
 
+    // INFO - GM - 2020/03/03 - add file breadcrumbs link if it exists
+    if (props.breadcrumbs.length === 4 && state.appOpenedType) breadcrumbsList.push(props.breadcrumbs[3])
```
Comment (Skylsmoi) [inline frontend/src/container/WorkspaceContent.jsx]: avoid usage of magic numbers.
If you have to use one, add comment explaining it.
"Magic numbers" are static int in code that don't explain themselves.
here, I see 4 and 3 (from breadcrumbs[3])

File: frontend_app_folder_advanced/src/container/FolderAdvanced.jsx [hunk 2]
```diff
@@ -81,15 +82,17 @@ class FolderAdvanced extends React.Component {
...
+    await this.loadContent()
+    this.buildBreadcrumbs()
   }
 
   componentDidUpdate (prevProps, prevState) {
     const { state } = this
 
     if (prevState.content.content_id !== state.content.content_id) {
       this.loadContent()
+      this.buildBreadcrumbs()
```
Comment (Skylsmoi) [inline frontend_app_folder_advanced/src/container/FolderAdvanced.jsx:95]: in componentDidMount, you added an await which you didn't add here. Is it intentional ?

Comment (gbonaspetti) [reply frontend_app_folder_advanced/src/container/FolderAdvanced.jsx:95]: no, actually, await is needed here too to ensure the correctness of the breadcrumbs. I will make the change and resolve this conversation.

**Comment by metzgegu** [comment]
Tested and verified on branch.

**Pull request merged**